### PR TITLE
Wallet card upgrade

### PR DESCRIPTION
**Pull request by wackerow**
## Description
- Refactors `WalletCard` component with ability to flip card over for more details.
- Upgrades some wallet logo images to better quality, with added transparency where needed, making them more consistent throughout the grid of wallets. Also added a white (#FFF) background to those with none. A couple had backgrounds that may be their "brand color" but did not fit correctly given their logo and our design (ie: Unstoppable.. made background black to fit their website theme)
- Button link added to take to project, since the entire card can't link there is we want to make it interactive.
- "Pointer" cursor and "flip" icon used on hover to indicate clickable regions of the card.

Should consider which features (ie `has_defi_integrations` etc) to show on the back, perhaps trimming some to save space.

I thought the emojis were a decent way to condense information on desktop/web/mobile/hardware. Planned to add a tool-tip here, but not in love with the current layout of these as of commit 7645767.

Note: `wallets.csv` has been updated manually, and these changes are not reflected in spreadsheet.

Feedback appreciated =) cc: @ryancreatescopy @samajammin

### CODE REVIEW

**Comment by ryancreatescopy** [comment]
This was an amazing start. I pushed a commit with some proposed changes:
- Made it clearer that there was more info with a button
- reduced vertical height with a new title/image area
- moved wallet type to front of card and added text labels (emojis alone weren't clear)  – I feel like this is a good immediate differentiator.
 
Some wallet images didn't work in this new layout so we'll need to fix that...

File: src/components/WalletCard.js [hunk 1]
```diff
@@ -58,21 +103,138 @@ const Title = styled.h3`
...
+      rotateY: 180,
+      transition: {
+        duration,
+      },
+    },
+  }
+
+  const getFeatureStatus = (feature) =>
+    feature === "TRUE" ? (
+      <Emoji size="0.75rem" text="❌" />
```
Comment (ryancreatescopy) [inline src/components/WalletCard.js:218]: Do we need to give these aria-labels to represent yes/no?

Comment (wackerow) [reply src/components/WalletCard.js:218]: Hm, we haven't in the past for other Emoji usage, though we aren't usually using them to actually convey data. Probably a good idea here

**Comment by wackerow** [comment]
@ryancreatescopy Love the changes =) Will look into the images, and will think of good way to filter between web/mobile/desktop/hardware.

**Comment by wackerow** [comment]
@samajammin cleared up some merge conflicts. Ready for another (final?) review

**Review by samajammin — APPROVED** [review]
Great work @wackerow! Did some minor refactoring.

**Pull request merged**